File: Desarrollos AddOn SAP B1/Samples/COM DI/VB.NET/17.Installments/Installment.vb

'****************************************************************************
'  SAP MANAGE DI API 2007 SDK Sample
'****************************************************************************
'
'  Copyright (c) SAP MANAGE
'
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
' ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
' THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
' PARTICULAR PURPOSE.
'
'****************************************************************************
' In this sample the SAP Business one order form is imitated via DI
'****************************************************************************

Module DownPayment
    Public oRecordSet As SAPbobsCOM.Recordset ' A recordset object

    '****************************************************************************
    ' This is the application entry point
    '****************************************************************************
    Public Sub Main()
        Dim fMain As frmMain ' The main form object
        fMain = New frmMain ' allocating a new form
        fMain.ShowDialog() ' showing the form
        fMain.Dispose() ' destroy the form object
        fMain = Nothing
    End Sub
End Module

File: Desarrollos AddOn SAP B1/Samples/COM DI/CSharp/17.Installments/Installment.cs
using System.Diagnostics;
using Microsoft.VisualBasic;
using System;
using System.Data;
using System.Drawing;
using System.Windows.Forms;
using System.Collections;


//****************************************************************************
//  SAP MANAGE DI API 2007 SDK Sample
//****************************************************************************
//
//  Copyright (c) SAP MANAGE
//
// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
// PARTICULAR PURPOSE.
//
//****************************************************************************
// In this sample the SAP Business one order form is imitated via DI
//****************************************************************************

namespace BGT
{
	sealed class DownPayment
	{
		public static SAPbobsCOM.Recordset oRecordSet; // A recordset object
		
		//****************************************************************************
		// This is the application entry point
		//****************************************************************************
		public static void Main ()
		{
			frmMain fMain; // The main form object
			fMain = new frmMain(); // allocating a new form
			fMain.ShowDialog(); // showing the form
			fMain.Dispose(); // destroy the form object
			fMain = null;
		}
	}
	
}
